views/render-tracking-events.php

```diff
@@ -0,0 +1,7 @@
+<script type="text/javascript">
+    <?php
+        foreach($this->events_queue as $event) {
+            echo html_entity_decode($event);
+        }
+    ?>
```

Comment:
Same. Use a safe javascript to guard agains not-yet-loaded Metrilo javascript library.
```javascript
  (function() {
    const f = function () {
      <?php
          foreach($this->events_queue as $event) {
              echo html_entity_decode($event);
          }
      ?>
    };
    if (typeof window.metrilo !== 'undefined') {
      f();
    } else {
      window.metriloQueue = window.metriloQueue || [];
      window.metriloQueue.push(f);
    }
  })();
```